modules/meta/arrow-meta-prototype/idea-plugin/src/main/kotlin/arrow/meta/dsl/ide/editor/quickFix/QuickFixSyntax.kt

```diff
@@ -0,0 +1,54 @@
+package arrow.meta.dsl.ide.editor.quickFix
+
+import com.intellij.codeInsight.intention.IntentionAction
+import com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.diagnostics.DiagnosticFactory
+import org.jetbrains.kotlin.idea.quickfix.KotlinIntentionActionsFactory
+import org.jetbrains.kotlin.idea.quickfix.KotlinQuickFixAction
+import org.jetbrains.kotlin.idea.quickfix.QuickFixContributor
+import org.jetbrains.kotlin.idea.quickfix.QuickFixes
+
+interface QuickFixSyntax : QuickFixUtilitySyntax {
```

Comment:
This guy is still WIP, But each Syntax, which is not ready, yet is not included in the IdeSyntax.